pyanaconda/ui/webui/__init__.py

```diff
@@ -118,7 +102,7 @@ def run(self):
 
         proc = startProgram(["/usr/libexec/webui-desktop",
                             "/cockpit/@localhost/anaconda-webui/index.html",
-                            profile_name],
+                            profile_name, str(int(self.remote))],
```

Comment:
I think it would be a good idea to add a docstring explaining whats going on here. :) (eq. that the script being called getting reacts differently on 1/0 being passed)

Reply:
Parameters are not named and explained in the script.

Reply:
OK, that should be good enough. :)